### PR TITLE
Add the GMT/PyGMT preworkshop at AGU24 to the "External resources" section

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

Add the GMT/PyGMT preworkshop at AGU24 to the "External resources" section:

  - [ ] Create and add thumbnail image (similar to the egu2022 workshop, see #1941)

**Preview**: https://pygmt-dev--3689.org.readthedocs.build/en/3689/external_resources.html

**Fixes**: #3683 (partly)

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash command is:

- `/format`: automatically format and lint the code

### CODE REVIEW

**Comment by seisman** [comment]
@yvonnefroehlich It seems the thumbnail image for the "PyGMT-Howto" tutorial is also broken. Please help fix it in this PR.

**Comment by yvonnefroehlich** [comment]
> @yvonnefroehlich It seems the thumbnail image for the "PyGMT-Howto" tutorial is also broken. Please help fix it in this PR.

The URL is updated in commit https://github.com/GenericMappingTools/pygmt/pull/3689/commits/cdcc26ee7a41dea0f3d217ad34a8857667edaa59.

**Comment by yvonnefroehlich** [comment]
thumbnail_preworkshop

**Comment by seisman** [comment]
I just downloaded the image and upload it again. Here is the link I have:

https://github.com/user-attachments/assets/f92e1391-78f4-41bc-8ed8-5df1f59b8362

**Review by seisman — APPROVED** [review]
Looks good to me.

File: doc/external_resources.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ to submit a pull request with your recommended addition to the
 
 :::::{grid} 1 2 2 3
 
+::::{grid-item-card} 2024 AGU PREWS9: Mastering Geospatial Visualizations with GMT/PyGMT
+:link: https://github.com/GenericMappingTools/agu24workshop
+:text-align: center
+:margin: 0 3 0 0
+
+![](https://github.com/user-attachments/assets/f92e1391-78f4-41bc-8ed8-5df1f59b8362)
```
Comment (weiji14) [inline doc/external_resources.md:20]: Could we have a slightly taller screenshot?

```suggestion
![](https://github.com/user-attachments/assets/9f3ab0ed-83f1-4cff-bc68-387fc13ca61f)
```

image

**Review by weiji14 — APPROVED** [review]
Thanks for adding this @yvonnefroehlich!

File: doc/external_resources.md (hunk 1 above)
Comment (yvonnefroehlich) [reply doc/external_resources.md:20]: I also prefer to show the figure of Tutorial 4 completely, but there is already quite a lot white space when using the current screenshot in the neighboring cards, so I was unsure using a taller screenshot. But let's try and see how it looks.

Comment (seisman) [reply doc/external_resources.md]: What about this one? The title is removed from the screenshot.
Screenshot from 2024-12-21 12-12-25

`https://github.com/user-attachments/assets/b86fd4c1-c5dd-4d51-a7fe-362d86cc01b0`

Comment (yvonnefroehlich) [reply doc/external_resources.md]: Actually, I thought about not including the title in the screenshot, but kept it to be consistent with the image for the EGU2022 workshop. But let's see how it looks after https://github.com/GenericMappingTools/pygmt/pull/3689/commits/2e8b38ef1a0357470282628584d57982d878b8ca.

Comment (seisman) [reply doc/external_resources.md]: I think you can choose anyone you like, then merge this PR.

**Pull request merged**